static/app/views/issueList/searchBar.tsx

```diff
@@ -93,6 +131,10 @@ const EXCLUDED_TAGS = ['environment'];
 function IssueListSearchBar({organization, tags, ...props}: Props) {
   const api = useApi();
   const {selection: pageFilters} = usePageFilters();
+  const {tags: issueTags} = useFetchIssueOrganizationTags({
+    orgSlug: organization.slug,
+    projectIds: pageFilters.projects.map(id => id.toString()),
```

Comment:
We should pass the option `enabled: organization.features.includes('...')` here so this request only gets made if the feature flag is enabled. You could also add a wrapper component for `SearchQueryBuilder` and add the hook there so it only gets called if that component is rendered.